src/controllers/groupe.controller.ts

```diff
@@ -0,0 +1,83 @@
+import { NextFunction, Request, Response } from 'express';
+import { Group } from '@prisma/client';
+import { CreateGroupDto } from '@dtos/group.dto';
+import groupService from '@services/groups.service';
```

Comment:
```suggestion
import GroupService from '@services/groups.service';
```
Typo Correction: In the import statement, change groupService to GroupService for consistency and clarity.